app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -92,6 +94,46 @@ class WalkthroughTopicListFragmentTest {
     )
   }
 
+  @Test
+  fun testWalkthroughWelcomeFragment_recyclerViewIndex1_topicHeader_notOnTop_progressBarIsHidden() {
+    launch<WalkthroughActivity>(createWalkthroughActivityIntent(0)).use {
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.walkthrough_welcome_next_button)).perform(scrollTo(), click())
+      onView(withId(R.id.walkthrough_topic_recycler_view)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          4
+        )
```

Comment:
Tests are working only when ```scrollToPosition()``` is passed with the parameter >= 3, tests should pass for parameter >= 1

Probably if the item is already visible, then ```scrollToPosition()``` does not work.